### ISSUE TITLE
The performance difference between the validation set and the test validation set

### ISSUE THREAD

**Issue by sunguwei** [baidut/paq2piq#5]
In your released paper, I find that the performance difference between the validation set and the test validation set is quite large, which is not normal. Could you explain why?

**Comment by baidut** [comment]
Thank you! Good question. The test set contains images bigger than 640x640 while the validation set contains images that have a width or height smaller than 640 and have similar size distribution with the training set.
